src/switcher-ui.tsx

```
@@ -1157,6 +1171,8 @@ function SwitcherApp() {
                 const idx = selectedSessionIndex >= 0 ? selectedSessionIndex : 0;
                 const s = sessions[idx];
                 if (s) {
+                  // Arm before opening, in case the bridge triggers the focus cycle synchronously.
```

ESLint

[error] Insert `····`

(prettier/prettier)